PowerNSX.psm1

```diff
@@ -19871,11 +19874,16 @@ function New-NsxSecurityTag {
         Add-XmlElement -xmlRoot $xmlnodes -xmlElementName "typeName" -xmlElementText "SecurityTag"
         Add-XmlElement -xmlRoot $xmlRoot -xmlElementName "name" -xmlElementText $Name
 
+
         #Optional fields
         if ( $PsBoundParameters.ContainsKey('Description')) {
             Add-XmlElement -xmlRoot $xmlRoot -xmlElementName "description" -xmlElementText "$Description"
         }
 
+        if ( $PsBoundParameters.ContainsKey('Universal')) {
+            Add-XmlElement -xmlRoot $xmlRoot -xmlElementName "isUniversal" -xmlElementText $Universal
```

Comment:
Not that I am aware of. Since we aren't passing a string in the parameters and the XML requires it to be isUniversal. when sent.